uamqp/authentication/cbs_auth_async.py

```diff
@@ -46,13 +46,12 @@ async def create_authenticator_async(self, connection, debug=False, loop=None, *
         :param debug: Whether to emit network trace logging events for the
          CBS session. Default is `False`. Logging events are set at INFO level.
         :type debug: bool
-        :param loop: A user specified event loop.
-        :type loop: ~asycnio.AbstractEventLoop
         :rtype: uamqp.c_uamqp.CBSTokenAuth
         """
-        self.loop = loop or get_running_loop()
+        self._internal_kwargs = get_dict_with_loop_if_needed(loop)
```

Comment:
same comment as before on the default get_running_loop() behavior for python 3.6 -> 3.9